core/trino-parser/src/main/java/io/trino/sql/SqlFormatter.java

```diff
@@ -1282,7 +1312,7 @@ private static String formatPrincipal(PrincipalSpecification principal)
         }
 
         @Override
-        protected Void visitDropTable(DropTable node, Integer context)
+        protected Void visitDropTable(DropTable node, Integer indent)
```

Comment:
`checkArgument`? (and below)

Reply:
Yeah. Sequence of `private` methods fooled me. We are already past public interface.